source/common/http/headers.h

```diff
@@ -91,6 +91,7 @@ class HeaderValues {
   const LowerCaseString EnvoyNotForwarded{absl::StrCat(prefix(), "-not-forwarded")};
   const LowerCaseString EnvoyOriginalDstHost{absl::StrCat(prefix(), "-original-dst-host")};
   const LowerCaseString EnvoyOriginalPath{absl::StrCat(prefix(), "-original-path")};
+  const LowerCaseString EnvoyOriginalPathWithDoubleSlash{absl::StrCat(prefix(), "-original-path-with-double-slash")};
```

Comment:
This could be "triple-slash" etc. Is there a better/slightly more generic definition, "unmerged-slashes" perhaps?

Reply:
sounds good to me. how about envoy-original-path-unmerged-slashes?

Reply:
sounds good!